views/js/qtiCommonRenderer/renderers/interactions/ChoiceInteraction.js

```diff
@@ -92,11 +92,12 @@ define([
 
         var min = interaction.attr('minChoices'),
             max = interaction.attr('maxChoices'),
+            msg,
             choiceCount = _.size(interaction.getChoices()),
             minInstructionSet = false;
 
-        //if maxChoice = 1, use the radio gorup behaviour
-        //if maxChoice = 0, inifinite choice possible
+        //if maxChoice = 1, use the radio group behaviour
+        //if maxChoice = 0, infinite choice possible
         if(max > 1 && max < choiceCount){
```

Comment:
I know it has already been like this before, but I think it should be max>=1, otherwise it can never reach the case 'You must select exactly 1 choice'.